Android/Io.OpenCensus/readme.md

```diff
@@ -0,0 +1,21 @@
+# OpenCensus 
+
+IO.OpenCensus bindings for Xamarin.Android which are Grpc dependencies (Grpc.Core).
+
+*   packages bound
+
+    *   `OpenCensusApi` (`io.opencensus-api`)
+
+        *   https://repo1.maven.org/maven2/io/opencensus/opencensus-api/0.12.3/opencensus-api-0.12.3.jar
```

Comment:
Might want to consider leaving out full url's here since it's another spot to update when the version is changed, and doesn't really have an impact here otherwise.

Reply:
This was merely while  I was searching for binaries.

You are using `search.maven.org`

https://search.maven.org/search?q=g:io.opencensus%20AND%20a:opencensus-contrib-grpc-metrics&core=gav

which does not list 0.12.3, while one of my repo search portals is

https://mvnrepository.com/artifact/io.opencensus/opencensus-contrib-grpc-metrics/0.12.3
and 0.12.3 list is there.

Reply:
Removed links

Reply:
OK. Version 0.12.3 found. - web UI has scroll behavior and that version was at the bootom not visible and scrollbar was also not visible

Reply:
Added generic links w/o version for future searches. (We'll need it soon)